docs/devices/YRD226_246_TSDB.md

```diff
@@ -28,6 +35,12 @@ To control this lock publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set`
 To read the current state of this lock publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"state": ""}`.
 This lock exposes a lock state which can be found in the published state under the `lock_state` property. It's not possible to read (`/get`) or write (`/set`) this value. The possible values are: `not_fully_locked`, `locked`, `unlocked`.
 
+### User Status
```

Comment:
This part of the documentation is generated, in order to make it show up here please add it to the `exposes` of this device (in zigbee-herdsman-converters)

Reply:
Where is this generated? I assumed that there would be some templating going on somewhere (e.g. this markdown file would not include anything in the 'Exposes' section) but I can't see where it's happening.

Reply:
Maybe another way of asking - is this markdown file pre or post generation? @Koenkk

Reply:
Exposes part is generated from https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docgen/device_page_exposes.js , everything under notes comes from https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docgen/device_page_notes.js

To regen the documentation execute `npm run docgen`

Reply:
@Koenkk is this now fixed as you made it expose 'pin_code' in the device definition?

Reply:
@jpg0 yes, so everything except  Device type specific configuration can be removed from this pr (other things will be generated with next release)

Reply:
Is possible to expand this functionality to YALE YMF40?